src/main/java/net/sf/jabref/gui/dbproperties/FieldFormatterCleanupsPanel.java

```diff
@@ -44,7 +44,7 @@
 
     private JButton resetButton;
 
-    private FieldFormatterCleanups defaultFormatters;
+    private final FieldFormatterCleanups defaultFormatters;
```

Comment:
Could you please exclude this file from your commit as I am currently working with this file for #957  and #958?
I included your changes already.